resources/core/google/app-engine.ts

```diff
@@ -1,10 +1,15 @@
 import * as gcp from '@pulumi/gcp';
+import * as pulumi from '@pulumi/pulumi';
 import { provider } from './provider';
 import { project } from './project';
 import { apiServices } from '../../google/api-services';
 import { apiServices as localApiServices } from './api-services';
 import { appEngineLocation } from '../config';
 
+export const serviceAccount = pulumi.output(
+  gcp.appengine.getDefaultServiceAccount(),
```

Comment:
`getDefaultServiceAccount` returns a `Promise`, and to use it more _natively_ in Pulumi I'm using `output` to convert it to a `Pulumi.Output` object.